### DISCUSSION TITLE
Laravel Nova 4 support

### DISCUSSION THREAD

**Discussion by wamesro** [digitlimit/alert#39] [Ideas]
Is it possible to use these Alerts in Laravel Nova 4 as well?

If so how?

**Comment by digitlimit** [comment]
Hello @wamesro, I will be running a test in Laravel Nova and will keep you updated.

**Reply by digitlimit** [reply]
@wamesro I have created an issue for this. [Issue](https://github.com/digitlimit/alert/issues/45)
